build/python/backend/Dockerfile

```diff
@@ -7,14 +7,6 @@ ARG YARA_PYTHON_VERSION=4.2.0
 ARG CAPA_VERSION=3.2.1
 ARG EXIFTOOL_VERSION=12.44
 
-RUN apt-get update -y && apt-get install -y \
```

Comment:
Assuming these versions are shipped with the new ubuntu?

Reply:
I think so, they failed to install. There could be more upgrades I need to do on top which I'll see when published.